### PR TITLE
Improved test speed

### PR DESCRIPTION
**Pull request by SamMousa**
This PR improves test speed by skipping framework tests if we're certain that changes don't impact them

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -72,6 +72,7 @@ services:
   - redis
 
 before_install:
+  - '$(php PruneTest.php'
```
Comment (samdark) [inline .travis.yml]: Looks weird. What `$(` means?

Comment (samdark) [reply .travis.yml]: Where's the closing `)`?

Comment (SamMousa) [reply .travis.yml]: Hmm there's a missing brace :|
The script will for example this:
```
export FRAMEWORK=skip
```

The `$(...)` makes bash interpret that and actually changes the environment.

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request merged**